CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 Change log
 -----------
 
+* (backport) kernel-modules-headers: specify cross compiler command [Florin]
```

Comment:
I don't understand. You want the "(backport)" reference to be removed from the Changelog entry?